go-controller/hybrid-overlay/pkg/util/util.go

```diff
@@ -33,7 +34,11 @@ func ParseHybridOverlayHostSubnet(node *kapi.Node) (*net.IPNet, error) {
 // node which does not participate in the ovn-kubernetes overlay network
 func IsHybridOverlayNode(node *kapi.Node) bool {
 	if config.Kubernetes.NoHostSubnetNodes != nil {
-		nodeSelector, _ := metav1.LabelSelectorAsSelector(config.Kubernetes.NoHostSubnetNodes)
+		nodeSelector, err := metav1.LabelSelectorAsSelector(config.Kubernetes.NoHostSubnetNodes)
+		if err != nil {
+			klog.Errorf("NoHostSubnetNodes label selector is not valid: %v", err)
```

Comment:
nit: add node info in the log so that there is more information for which node you are talking about..

Reply:
I moved the conversion so it is done once.